### ISSUE TITLE
Is this still being worked on?

### ISSUE THREAD

**Issue by CCF100** [PonyKart/PonyKart-UE4#2]
Just curious...

**Comment by lapsio** [comment]
Yep. Wanna help? xD

**Comment by CCF100** [comment]
Still learning to code unfortunately...

**Comment by lapsio** [comment]
yeah, thanks for website issue report. It worked fine yesterday...

**Comment by LowPolyFier** [comment]
Is this still being worked on in 2021? I personally don't think it is (last update was a year ago and the website is down...) but hey, asking won't hurt.

**Comment by Kreijstal** [comment]
> Is this still being worked on in 2021? I personally don't think it is (last update was a year ago and the website is down...) but hey, asking won't hurt.

did you fork it :)
